src/cascadia/WindowsTerminal/main.cpp

```diff
@@ -184,16 +184,13 @@ int __stdcall wWinMain(HINSTANCE, HINSTANCE, LPWSTR, int)
             }
         }
 
-        // GH#7125 = System XAML will show a system dialog on Alt Space. We might want to
-        // explicitly prevent that - for example when an action is bound to it. So similar to
-        // above, we steal the event and hand it off to the host. When the host does not process
-        // it, we will still dispatch like normal.
+        // GH#7125 = System XAML will show a system dialog on Alt Space. We want to
+        // explicitly prevent that because we handle that ourselves. So similar to
+        // above, we steal the event and hand it off to the host.
         if (_messageIsAltSpaceKeypress(message))
         {
-            if (host.OnDirectKeyEvent(VK_SPACE, LOBYTE(HIWORD(message.lParam)), true))
-            {
-                continue;
-            }
+            host.OnDirectKeyEvent(VK_SPACE, LOBYTE(HIWORD(message.lParam)), true);
+            continue;

```

Comment:
huh, why don't we need the `if()` here anymore?

Reply:
We want to always consume the `alt+space` since this is now a default keybinding (so if the host did not deal with it, it means it was explcitly unbound and we still don't want system XAML to get it)